jobqueue.go

```diff
@@ -58,15 +58,16 @@ type Status struct {
 // Using a stack for job processing can be a good way to protect an application from
 // bursts of chatty clients or temporarily slow job execution.
 type Stack struct {
-	options Options
-	stack   *stack
-	req     chan *job
-	done    chan struct{}
-	quit    chan bool
-	closing bool
-	status  chan chan Status
-	hasQuit chan struct{}
-	busy    int
+	options   Options
+	stack     *stack
+	req       chan *job
+	done      chan struct{}
+	quit      chan bool
+	closing   bool
+	status    chan chan Status
+	configure chan Options
```

Comment:
maybe configure -> reconfigure
because we use it only in the case of reconfiguration not the initial configuration.

Reply:
done, renamed